pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -157,19 +157,13 @@ func (r RunRepository) ArchiveBatch(ctx context.Context, ids []string) error {
 // Delete removes the existing models.Run from the db.
 func (r RunRepository) Delete(ctx context.Context, run *models.Run) error {
 	if err := r.db.Transaction(func(tx *gorm.DB) error {
-		// find the row_num for the run
-		minRowNum, err := r.getMinRowNum(tx, []string{run.ID})
-		if err != nil {
-			return err
-		}
-
 		// delete the row
-		if err := tx.Model(&run).Delete(run).Error; err != nil {
+		if err := tx.Clauses(clause.Returning{}).Model(&run).Delete(&run).Error; err != nil {
```

Comment:
could you please narrow down the `Returning` clause to only the `row_num` column?